mbed/mbed.py

```diff
@@ -1599,13 +1599,19 @@ def _find_file_paths(self, paths, fl):
         return None
 
     def requirements_contains(self, library_name):
-        req_path = self.get_requirements() or self.path
+        req_path = self.get_requirements()
+        if not req_path:
+            return None
```

Comment:
So this function can now return None, False, or True? What are these values used for?

Reply:
Good point!